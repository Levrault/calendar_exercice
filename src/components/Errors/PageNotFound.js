import React from 'react';

/**
 * Page not found component
 * @returns {node}
 */
const PageNotFound = () => (
  <div>
    404 not found
  </div>
);

export default PageNotFound;
