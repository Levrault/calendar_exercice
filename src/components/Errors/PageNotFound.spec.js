import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from './PageNotFound';

describe('PageNotFound component', () => {
  it('should render without problem', () => {
    const component = renderer.create(
      <PageNotFound />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
