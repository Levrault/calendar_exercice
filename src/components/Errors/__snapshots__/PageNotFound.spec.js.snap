// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component should render without problem 1`] = `
<div>
  404 not found
</div>
`;
